monad/README.md

```diff
@@ -3,7 +3,7 @@ title: Monad
 category: Functional
 language: en
 tag:
- - Reactive
+- Reactive
```

Comment:
Avoid removing spaces. They are meant for indentation

Reply:
I'm sorry, I have added the space back